lib/models/introductory_price.freezed.dart

```diff
@@ -61,18 +57,13 @@ abstract class $IntroductoryPriceCopyWith<$Res> {
       _$IntroductoryPriceCopyWithImpl<$Res, IntroductoryPrice>;
   @useResult
   $Res call(
-      {@JsonKey(name: 'price')
-          double price,
-      @JsonKey(name: 'priceString')
-          String priceString,
-      @JsonKey(name: 'period')
-          String period,
-      @JsonKey(name: 'cycles')
-          int cycles,
+      {double price,
+      String priceString,
+      String period,
+      int cycles,
       @JsonKey(name: 'periodUnit', unknownEnumValue: PeriodUnit.unknown)
```

Comment:
I think this will probably break the score too. But I doubt we can control adding the ignore here since this is autogenerated.

Reply:
Yeah... Maybe the rules in pub.dev ignore freezed file? But not sure... I tried to remove these but it didn't work well. Wanted to get the "easier" issues fixed and we can see how that affects the pub.dev score